examples/ucp_hello_world.c

```diff
@@ -262,16 +266,22 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
                                   recv_handler);
 
     if (UCS_PTR_IS_ERR(request)) {
-        fprintf(stderr, "unable to receive UCX data message (%u)\n",
-                UCS_PTR_STATUS(request));
-        free(msg);
-        goto err_ep;
+        status = UCS_PTR_STATUS(request);
```

Comment:
maybe move this error handling to `wait` function, so you will not need to duplicate the code thru this example?

Reply:
done